eternalcore-core/src/main/java/com/eternalcode/core/feature/jail/JailPermissionConstant.java

```diff
@@ -0,0 +1,48 @@
+package com.eternalcode.core.feature.jail;
+
+import com.eternalcode.annotations.scan.permission.PermissionDocs;
+
+public class JailPermissionConstant {
```

Comment:
Only permissions that don’t come from commands should be documented here. Command annotations generate documentation automatically. @PermissionDocs is needed only when a permission doesn’t belong to a command, for example in a listener.